tests/test_output_multi_poly_trivial.osm

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<osm version="0.6">
+<!--
+
+   1_____2     3_____4
+   |     |     |     |
+   |  w1 |     | w2  |
+   5_____6     7_____8
+
+this file defines the above nodes, ways and a relation
+which contains both w1 and w2.
+-->
+  <node id="1" lon="0" lat="1"/>
+  <node id="2" lon="1" lat="1"/>
+  <node id="3" lon="2" lat="1"/>
+  <node id="4" lon="3" lat="1"/>
+  <node id="5" lon="0" lat="0"/>
```

Comment:
0,0 shouldn't be used in tests since it's a special point to the dense backend (#158)